scripts/common.cake

```diff
@@ -293,7 +293,22 @@ public class BuildEnvironment
         }
         else
         {
-            return context.GitVersion();
+            try
+            {
+                return context.GitVersion();
+            }
+            catch
```

Comment:
Can we have this catch block only kick in when not running on CI?

Reply:
Sure, but currently when on CI, we run GitVersion as separate job, and get GitVersion stuff as env vars. So CI will never run GitVersion in this context.

Reply:
ah right